thingif/src/androidTest/java/com/kii/thingif/ThingIFAPI_PatchTriggerTest.java

```diff
@@ -37,8 +39,7 @@
  */
 @RunWith(AndroidJUnit4.class)
 public class ThingIFAPI_PatchTriggerTest extends ThingIFAPITestBase {
-    @Test
-    public void patchTriggerWithCommandTest() throws Exception {
+    private void _patchTriggerWithCommandTest(Predicate predicate) throws Exception {
```

Comment:
In java method name starting '_' is not so popular convention.

Can we choose different name?